src/chain/elders_info.rs

```diff
@@ -120,22 +136,19 @@ impl EldersInfo {
 
     /// Returns `true` if the proofs are from a quorum of this section.
     pub fn is_quorum(&self, proofs: &ProofSet) -> bool {
-        // WIP: the call to members is probably to heavy?
-        proofs
-            .ids()
-            .filter(|id| self.members().contains(id))
-            .count()
-            * QUORUM_DENOMINATOR
-            > self.members.len() * QUORUM_NUMERATOR
+        if proofs.len() * QUORUM_DENOMINATOR <= self.len() * QUORUM_NUMERATOR {
```

Comment:
I would add a comment here explaining why are we doing this check seemingly twice.

Reply:
Dropped this commit as local tests didn't show any performance improvements